docs/providers/documentation/graylog-provider.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Graylog Provider"
+sidebarTitle: "Graylog Provider"
+description: "The Graylog provider enables webhook installations for receiving alerts in KeepHQ"
+---
+
+## Overview
+
+The **Graylog Provider** facilitates receiving alerts from Graylog by setting up Webhook connections. It allows seamless integration with Graylog to receive notifications about events and alerts through KeepHQ.
+
+## Authentication Parameters
+
+- **graylogUserName**: Username for authenticating with Graylog's API.
+- **graylogAccessToken**: Access token for authenticating with Graylog's API.
+- **deploymentUrl**: Deployment URL for connecting to the Graylog instance (e.g., `http://localhost:9000`).
+
+## Scopes
+
+- **authenticated**: Mandatory for all operations, ensures the user is authenticated.
+- **authorized**: Mandatory for querying incidents and managing resources, ensures the user has admin privileges.
+
+## Connecting with the Provider
```

Comment:
the instructions is only for self hosted?

Reply:
Self Hosted.